### PR TITLE
Improve gofmt task

### PR DESCRIPTION
**Pull request by aledbf**
Running `make gofmt` now fixes the format and imports of go files

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,7 +25,7 @@ install:
 bindata:
 	which go-bindata > /dev/null || go get -u github.com/jteeuwen/go-bindata/...
 	rm -rf bootstrap/bootstrap.go
-	go-bindata -pkg bootstrap -o bootstrap/bootstrap.go bootstrap/ bootstrap/vpn
+	go-bindata -pkg bootstrap -o bootstrap/bootstrap.go bootstrap/
```
Comment (MyDigitalLife) [inline Makefile]: The bootstrap/vpn was removed here. This is needed to make sure the files from this folder can be read.

**Comment by kris-nova** [comment]
Hi @aledbf long time no see!

File: Makefile (hunk 1 above)
Comment (aledbf) [inline Makefile]: @MyDigitalLife sorry about that

**Comment by kris-nova** [comment]
LGTM - thanks for updating

**Pull request merged**